Sloth.Core/Configuration/WebHookServiceOptions.cs

```diff
@@ -0,0 +1,9 @@
+namespace Sloth.Core.Configuration
+{
+    public class WebHookServiceOptions
+    {
+        public int RetryDelaySeconds { get; set; } = 1;
+
+        public int MaxRetryDelaySeconds { get; set; } = 32;
```

Comment:
@sprucely As thins is a new Job... Maybe a setting so we can disable it?

Reply:
Working on another commit to honor the IsActive property on webhook